### PR TITLE
For #1105: Set title of a job in agenda from github

### PR DESCRIPTION
**Pull request by paulodamaso**
For #1105:

- Created tests for `set_agenda_title_from_github.groovy` 
- Created dummy implementation of `set_agenda_title_from_github.groovy` to pass the tests
- Added `title(Job)` method in `Agenda` for checking the title value
- Added `@todo` for definitive implementation of `set_agenda_title_from_github.groovy`

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#1132](https://github.com/zerocracy/farm/issues/1132) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#1132](https://github.com/zerocracy/farm/issues/1132) is assigned to @krzyk[/z](https://www.0crat.com/u/krzyk), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/1630443126); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: src/test/resources/com/zerocracy/bundles/set_agenda_title_from_github/_after.groovy [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+import com.jcabi.xml.XML
+import com.zerocracy.Farm
+import com.zerocracy.Project
+import com.zerocracy.pmo.Agenda
+import org.hamcrest.MatcherAssert
+import org.hamcrest.Matchers
+
+def exec(Project project, XML xml) {
+  Farm farm = binding.variables.farm
```
Comment (krzyk) [inline src/test/resources/com/zerocracy/bundles/set_agenda_title_from_github/_after.groovy]: @paulodamaso you can inline `farm` and `agenda`

File: src/test/resources/com/zerocracy/bundles/set_agenda_title_from_github/_before.groovy [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+import com.jcabi.github.Repos
+import com.jcabi.xml.XML
+import com.zerocracy.Farm
+import com.zerocracy.Project
+import com.zerocracy.entry.ExtGithub
+import com.zerocracy.pmo.Agenda
+
+def exec(Project project, XML xml) {
+  Farm farm = binding.variables.farm
```
Comment (krzyk) [inline src/test/resources/com/zerocracy/bundles/set_agenda_title_from_github/_before.groovy]: @paulodamaso you can inline farm

**Comment by paulodamaso** [comment]
@krzyk Thanks to the review, I've fixed the issues you have marked, but I'm having problems with things that I didn't even touched, is the build broken?

File: pom.xml [hunk 3]
```diff
@@ -585,6 +585,13 @@ SOFTWARE.
               </excludes>
               <license>file:${basedir}/LICENSE.txt</license>
             </configuration>
+      <executions>
```
Comment (krzyk) [inline pom.xml]: @paulodamaso here the lines should be indented differently, maybe that's why qulice complains (although in incorrect files)

**Comment by krzyk** [comment]
@paulodamaso why did you add `executions` inside pom?

**Comment by paulodamaso** [comment]
@krzyk Looks like we have something broken on master, other PRs (#1136, #1137, #1111 for example) are having the same errors; just donwloaded `master` in a fresh new git repo and I am having the same issues.

About `executions`, I was trying to run just `qulice` checks, but commited it unintentionally, sorry.

**Comment by g4s8** [comment]
@rultor merge

**Comment by paulodamaso** [comment]
@krzyk I really don't understand what's happening here, but at least  there is no new build failures in this PR. It is really strange that it's okay and then it suddenly breaks, or it's broken and then it turns ok. Master branch is showing the same problems, and again, it is happening a lot of PRs, maybe we should check it.

**Comment by krzyk** [comment]
@paulodamaso actually this error was correct, it just wasn't used in travis before only in rultor

**Comment by krzyk** [comment]
@g4s8 good to merge

**Comment by paulodamaso** [comment]
@g4s8 please take a look

**Comment by g4s8** [comment]
@rultor merge

**Comment by g4s8** [comment]
@rultor merge

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @krzyk[/z](https://www.0crat.com/u/krzyk), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#1132](https://github.com/zerocracy/farm/issues/1132) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by ypshenychka** [comment]
@0crat quality good

**Comment by 0crat** [comment]
Order was finished, quality is "good": +20 point(s) just awarded to @krzyk[/z](https://www.0crat.com/u/krzyk)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Pull request merged**